docs/js/init.js

```diff
@@ -236,14 +236,17 @@
 function copyText() {
   const copiedText = document.getElementById('copiedText').textContent
   const textArea = document.createElement('textArea');
+  const copyBtn = document.getElementById("copyButton");
+
   textArea.textContent = copiedText;
   document.body.append(textArea);
   textArea.select();
   textArea.setSelectionRange(0, 99999)
   document.execCommand('copy');
-  document.getElementById('copyButton')
-  .insertAdjacentHTML('afterend',
-  `<span style="margin-left:10px; font-size:14px;">Copied!</span>`
+  copyBtn.insertAdjacentHTML('afterend', 
+    `<span style="margin-left:10px; font-size:14px;">Copied!</span>`
   )
-  // alert("Code copied to clipboard: " + textArea.value);
-  }
\ No newline at end of file
+  copyBtn.classList.remove("waves-effect");
+  copyBtn.classList.remove("waves-light");
+  copyBtn.classList.add("disabled");
```

Comment:
I think you can just add `disabled` class. No need to remove the waves class. But overall it looks good to me

Reply:
all set :)